### DISCUSSION TITLE
Welcome to postgresql_cluster Discussions!

### DISCUSSION THREAD

**Discussion by vitabaks** [autobase-tech/autobase#76] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pavelkogen** [comment]
Hi! 
Patroni service does not start. The deployment of the role ends at the moment:
TASK [patroni : Wait for port 8008 to become open on the host] *******************************************************************************fatal: [X.X.X.X]: FAILED! => {"changed": false, "elapsed": 120, "msg": "Timeout when waiting for X.X.X.X:8008"}

It would be nice to include in the patroni role the ability to enable logging.

**Reply by vitabaks** [reply]
Hi @pavelkogen 
> Patroni service does not start. 

See postgres and patroni `sudo journalctl -u patroni` logs.


If you have any problems, do not hesitate to open a new [issue](https://github.com/vitabaks/postgresql_cluster/issues) :)
